src/FirstSteps/RPG/Items/OldChest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace FirstSteps.RPG.Items
+{
+    class OldChest
+    {
+        private static List<Item> itemsForHuman = new List<Item>()
+        {
+            new Pitchfork(),
+            new Knife()
+        };
+        public static Item Open()
+        {
+            var random = new Random().Next(0,5);
+            if(random > itemsForHuman.Count())
+            {
+                Console.WriteLine("\nUnfortunately, Empty slot!");
+                return null;              
+            }
+            else if(random == 0)
```

Comment:
It's a pretty naive approach because of how you will handle situations with three or more items on the list.

Reply:
Try to use the random number as an index of the list.